src/test/java/com/auth0/AuthenticationControllerTest.java

```diff
@@ -437,7 +437,7 @@ public void shouldSetSameSiteNoneCookiesAndNoLegacyCookieWhenIdTokenResponse() {
     public void shouldCheckSessionFallbackWhenHandleCalledWithRequestAndResponse() throws Exception {
         AuthenticationController controller = builderSpy.withResponseType("code").build();
 
-        AuthRequest codeExchangeRequest = mock(AuthRequest.class);
+        TokenRequest codeExchangeRequest = mock(TokenRequest.class);
```

Comment:
These updates are required so that when we mock the return of the `codeExchangeRequest`, it needs to return a `TokenRequest`.